src/sentry/slug/validators.py

```diff
@@ -0,0 +1,14 @@
+from django.core.validators import RegexValidator
+
+from sentry.slug.errors import DEFAULT_SLUG_ERROR_MESSAGE
+from sentry.slug.patterns import MIXED_SLUG_REGEX, ORG_SLUG_REGEX
+
+no_numeric_validator = RegexValidator(
+    regex=MIXED_SLUG_REGEX,
+    message=DEFAULT_SLUG_ERROR_MESSAGE,
+)
+
+org_slug_validator = RegexValidator(
+    regex=ORG_SLUG_REGEX,
+    message=DEFAULT_SLUG_ERROR_MESSAGE,
```

Comment:
This error message here should match the different regex, it's prob best to consolidate the error message here in `src/sentry/slug/errors.py`:
https://github.com/getsentry/sentry/blob/e7e6d3909f1074520c8a4f57f2d8463d98ba0794/src/sentry/api/serializers/models/organization.py#L99-L114

Reply:
We should probably share the messages across the two different ingress points then (API/DB) if we want to be consistent. This would require changes to the API field file, are we okay with that?

Reply:
yup that should be ok

Reply:
Instead of changing too many files, I consolidated the logic in the logic itself

Reply:
it was just one file, updated